src/components/dongheeKim/feeds.js

```diff
@@ -0,0 +1,223 @@
+import React, { Component } from 'react';
+import Comment from './comment';
+
+class Feeds extends Component {
+  constructor(feed) {
+    super();
+    this.state = {
+      feed,
+      value: '',
+    };
+  }
+
+  //댓글 내용 저장 함수
+  getComment = event => {
+    this.setState({
+      value: event.target.value,
+    });
+  };
+
+  //버튼으로 댓글 생성 함수
+  createComment = e => {
+    const comment = {
+      id: Date.now(),
+      accountName: 'donghee',
+      comment: this.state.value,
+    };
+
+    this.setState({
+      feed: { comments: this.state.feed.comments.concat(comment) },
+      value: '',
+    });
+  };
+
+  //엔터로 댓글 생성 함수
+  createCommentEnter = e => {
+    if (e.key === 'Enter') {
+      this.createComment();
+    }
+  };
+
+  //댓글 삭제 함수
+  deleteComment = comment => {
+    const newComments = this.state.feed.comments.filter(
+      item => item.id !== comment.id
```

Comment:
item이라는 변수명이랑 comment 라는 변수명이 혼동스럽네요
매개변수를 deleteComment로 받거나 또는 매개변수에서 comment 전체가 아닌 id값을 받는 식으로 변경해보는게 좋아보이네요!
```suggestion
      comment => comment.id !== deleteId
```

Reply:
연욱님~ 매개변수로 deleteComment를 설정하는 것이 더 좋을 것 같아 반영했습니다!